components/brave_wallet/browser/solana_tx_manager_unittest.cc

```diff
@@ -1537,4 +1537,18 @@ TEST_F(SolanaTxManagerUnitTest, RebroadcastTransaction) {
A57ejM9y8914zGCtbDJRkevDwH6ZyN+n7ZYAaOfXl2SY9PC5cOekuBgoa6V2fTzJPiJivRmEAezl3exBLTfH6DZKvdlyVgdBLNMazhYk4o/Z2bsIBj/q3vZSIXdeLNZmNAaxYBhFTiu4ZF6hRgrkCw0EJP3D5plUElHdMBKsKoNbYpNQExH0IgKXTpLg/W+bkpJmxxutq3HrumCgofwFeasCTSgl7BJvIX6HKxvFQVfKDEyuPM7WdA590rAbjmTDWDdnVcvyfPQQkXfFniw09RLkKG2nZG2DISmTt9z/fEfLxNY3AosikiVIvOfd8UJiWS7LS5OUCsczLIfMFYnfbRfVkZI/sYY9Ngs5320WY6UpTnQmLNqra7vASdiwz3ac2v8WxJG3rbmxc4I2hS2rsaUIQrPGaO3jCp6bDpWnqRv3HY34mgpWW5HGfYsVL7+YjAQ+BFWKor8ApvmbhicrhMK+7NJt6/qz7dbJhobQhggAtSztVXB7w9avZfh/kwPZ3Nd5Iao40zveSdPrwdDRoplLy7UbEotMsjv9Qwnz02xcvnCMpTV8C4LSRC5t13VjEi8IdJ8+LMEElU2onwi+z0Fol2ECdJcBZN/3DWHgtPP0+HYIJXYijesQtlDxywII4Pm1I9TIRZeZkfhBLL9Vgxut5MqDS2r82vxFwaeoIupgT1yFSvlz+h0dJ64i6pz3D/Hm0DosTjqRbAoi9m+QPucEcUg0wN7z6icWht2NnIBTG0zwEvjxjsAqrhbAb0BLUkePGkWzHNYmPtCtRc7jlN1cTEBdN+DW8a6UxVPf22yyXWeCdTlA4jLdADaK4T8EAAAAAAAevGKBgvk/En8hoiFg8piQmmnOGQxNa8OiBuzHLM/IOgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAArTIotnb3081ChKVEPxfxlis25JGzCkCyQFhJ5Ze6X7W9DuLSeDZLQ9AddxgPqlwvr56SGPOT+38EBrgYFS3ZeNRp2ZjKc541cDdBa4t7+JDifG3IwumGeqTIlY9pUyyr5Ib2o2LjkKl70X37jfQvWadk08Llc6CZNrdmMiM5IQSSa04Msx0B3HXoN5kGzIgmw3luoDY4nYYkrVFJPL0E5EEYytl0zJMTUbWIRBzpVlz9zoZ3dpkM5Gs95YStfuZJyclsfVnvjnf0Emm7gik1hzaRIw56tchwWKAbDb7xD4Wz61pDoa+T46cdwjK5ZmMiUoHbO8on8rNjZ5oIkDo7X6UBkCT0JVFQOHL7tCatUszyjK1vYv2UmYomMz2oydjyj6gyfjlAykKUlOKLVaf/ZoYbpo8nPfZ272VUPs2X4kpzJchgNmtwP8ljZmeloTx48Ja0BnYWlBS+YkcSnu+WwHxAa94p2dxmz6vvE0QZM5J9hf7N6cMcoVVVdEF0gQn11vMugoYLzpA4UXyFz34B9pgTJ/5rU4qdr5uAnidqxp6PxW7WMn0aLiqocFp3shNWtuvJrwnUwNPhc4Q9d9PvjpLYfw/beOq4ep20r/HL4y/Bvc73kbZgdOQMRkWKmtNaxwvp8lxONYpQ6LcCi52a9GnTCpl2LwLWTm3pNdhBPTvSySYM96EBAdiRYNGctX8ZWS86z7A70MS8dO9hwZktaZcEd2Qa+mRvQfR7c/+7jo9INyzXYIJAbzOJAm3S9ulsdjHiqQX/91C9nVd5w7tRClwTLxDqSi4l9iFW54vmEgbjQ6zCk4Y5E4R2/3hWexYk78l03C6PZktYaPxLDEWm9cqjL5m6Z241icmIYbIGrg4ZX4FiVJJcyLHpit2+Sd3wMOk/ZKMtC4qINKOTlKmLE1W06I6t35kCpdmBRtEvhzGXlRWB9QoXJc0t8cw2ZvQnBo3L6J/EvphikAgGY19UdZ3REwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACtMii2dvfTzUKEpUQ/F/GWKzbkkbMKQLJAWEnll7pftfo/yJGggownAFbIrnUZRv3uvcc0g9fzDMN9TfyWpcfvId25o1aBXD+sECa23sXfMSSvuttIXJulo+M5igS3uoXlh2mzKhvq8eonN1pECVoNH7Zkzi3TWOf8v7eMJqGTRH3elNXvEqX8Gynt5JInSEsZSmR/tcUWt1tx9YJedqGziHwivYdQ00AWrDxmtf8QLazdc/awFOcQtR6AIq+aGWjYg9MQnbGV4eIIkbIPcKFe6VkORtNlKBZst1IPCUxcThB5TBRdPHGtMYCjE1DcCIoV6OjP7/LaGscyIUITsqeLiGZbEKKYAG71zYrrp1oEDYFd3kwQ+h9JLkGqzq0BTw353D5/4BbgUO/yYDNPGKXU/jkdggkjGfWWTy4ut8HDpfixOknigvYJwxeoM/uNl20RUXxXHRIhomXSWvd47PiSNJDGzutFCuzcguKCkwMdEMfXO/heV78EGpc2CqLF2ZzB34LZxLh0E+ri7wSPlLTTVUzqc9krD3r5bgJxxpHiu2fJk1dKvOF8euO1ck0cM3DdVGj8GaqcsbM1EtdRTWes2nvOn06GGLa9L0EyznmM3Hpg5+FGCnKZ48Y0KleWJtInM+UPUm7C+hmiKzHo7VDyPNH9+UyRVO06dgmi8f+YH+HTtcgHsoHkaDzG1jFc+Vua3oZB3vyzI3LxwSbjmO96Wi3OCop/aLt0Vg+PcYN8LC67y/f/+0KuGJbxP3x0eaC0vbmnkLIoHDLfFW7KMwPFlkGzsKwzNXKnd5tJytO7pqurzT6xOUNgA4jrOIbOXNY0oSIECs9b7X5mkvcCCtD29EGFiO01okWM/+s5uT0m8Y0qsTvc5q7ljnuZNZ7C39lanBbcANbvGLeTOm+NxlzLVWZxOHdvfeoQEHDch5bjd034T0CuDIIp0NYGnlyPOafCmWd6CdNn/HsF47w4DuZSwFteMntjlly+/L8uGWH6kJfQZvJmU0i1VSdr1WI/SRqlQxgAAAAAAA==)");
```

Comment:
Can this blob be smaller? Where does it come from?
Maybe there are some public test vectors available?

Reply:
This is real data from the blockchain. First you fetch the merkle tree account for the compressed nft

```
curl --request GET \
     --url 'https://api.simplehash.com/api/v0/nfts/proof/solana/5A8bvmT9FaKmo12cuwcLSgvE6p7c6KrXvVUhxnHRT43m' \
     --header 'X-API-KEY: <key>' \
     --header 'accept: application/json' 
```

then you make the getAccountInfo rpc call
```
curl -X POST https://mainnet-beta-solana.brave.com/rpc -X POST -H "x-brave-key: <key>" -H "Content-Type: application/json" --data '{"jsonrpc": "2.0","id": 1,"method": "getAccountInfo","params": ["7eFJyb6UF4hQS7nSQaiy8Xpdq6V7Q1ZRjD3Lze11DZTd",{"encoding": "base64"}]}'
```

I couldn't find publically available test vectors outside of the real blockchain data. I supposed I could try to construct a shorter one manually, but it would be a little time consuming, and I think real data is probably better for the test.